etc/config.xml

```diff
@@ -42,6 +42,20 @@
                 <can_review_payment>1</can_review_payment>
                 <payment_action>authorize_capture</payment_action>
             </omise_offsite_alipay>
+
+            <omise_offsite_tesco>
+                <active>0</active>
+                <title>Tesco Bill Payment</title>
+                <model>OmiseTescoAdapter</model>
+                <is_gateway>1</is_gateway>
+                <can_initialize>1</can_initialize>
+                <can_use_checkout>1</can_use_checkout>
+                <can_capture>0</can_capture>
+                <can_review_payment>1</can_review_payment>
+                <payment_action>order</payment_action>
+                <allowspecific>1</allowspecific>
+                <specificcountry>FR</specificcountry>
```

Comment:
@jacekstan do we need this line?

Reply:
some old testing... sneaked in.. nice catch